monadic-rx/shared/src/main/scala/mhtml/rx.scala

```diff
@@ -306,8 +341,12 @@ object Var {
    */
   def create[A](initialValue: A)(register: Var[A] => Cancelable): Var[A] =
     new Var[A](Some(initialValue), register)
+
+
+  private[mhtml] lazy val isProduction: Boolean = Platform.checkIsProduction
```

Comment:
I searched for strings appearing in the `IllegalStateException` thrown from `addUpdateCall` inside of `monadic-rx/js/target/scala-2.12/monadic-rx-opt.js` and didn't see anything (compared to `monadic-rx/js/target/scala-2.12/monadic-rx-test-fastopt.js`, where I could easily see the entire function definition for `addUpdateCall`).